pangeo-cnes.tex

```diff
@@ -83,75 +98,89 @@ \subsection{Motivations, mission and goals}
 \begin{itemize}
-\item Satellite imagery analysis: a recent blog post\cite{b6} explains how to use Pangeo for processing and analysing in real time Landsat imagery. NASA has decided to fund Pangeo initiative for applying the ecosystem on remote sensing datasets that are being pushed on AWS.
+\item Satellite imagery analysis: a recent blog post\cite{b6} explains how to use Pangeo for processing and analysing in real time Landsat imagery. NASA has decided to fund Pangeo initiative for applying the ecosystem on remote sensing datasets that are being pushed on AWS\cite{b10}.
 \item Astronomy: several scientists are already using the web platform to explore Gaia DR2 data on GCE.
 \item Neuroscience: there is some on-going work to use Pangeo for analysis on human brain or electrophysiological datasets.
 \end{itemize}
 
-Such applications are most effective when deployed on cloud-optimized datasets. This means that data must be accessible in a cloud or distributed storage friendly format, like Zarr for multi-dimensional data, Cloud optimized Geotiff for satellite imagery, or Parquet for tabular data. See \cite{b7} for more details on this crucial point. Formats like NetCDF or CSV files are not adapted or optimized for use at scale.
+Applications combining large data sets from different sources are most effective when deployed on cloud infrastructure.  This means that data must be accessible in a cloud or distributed storage friendly format, like Zarr for multi-dimensional data, Cloud optimized Geotiff for satellite imagery, or Parquet for tabular data. See \cite{b7} for more details on this crucial point.
 
 \subsection{Ongoing work}
 \label{ssec:ongowork}
 
-The community is active and working on improving Pangeo possibilities and accessibility. This includes the following ongoing tasks:\begin{itemize}
-\item Active work for giving everyone the possibility to try Pangeo at scale with a simple clic. This is made possible by using Binder along with the ecosystem\cite{b8}.
-\item Community governance: the project is still young, it needs further organization. A lot of work was initiated on this in the last Pangeo developper meeting this summer.
-\item It has recently been decided to offer a specific cloud environment for the different scientific communities, in order to cope with each domain specific software stack.
-\item In order to achieve the above point, intensive work is currently being done on simplifying Cloud environment updates, by using modern tooling for Continous Deployment using Hubploy and CircleCI.
-\item As Pangeo is compatible not only with cloud but with different infrastructure, several Dask subcomponents share some logic for deploying clusters. A work to rationalize this logic and separate resources management from Dask scheduling is also ongoing.
+Advancing both, the community and the technological stack, is currently done along the following lines:
```

Comment:
I'm not an english expert, but this sentence kind of sounds wrong to me.

Reply:
Changed.